### DISCUSSION TITLE
set diskID when harvesting

### DISCUSSION THREAD

**Discussion by mikehenselersimfront** [wixtoolset/issues#7754] [Questions]
I want to harvest a multiple folders and package them into different cabs.
I am using heat from cmd line

I tried setting the DiskID on a directory created in the parent project (that i set as a directory reference in heat cmd), but still the harvested files go into cab1 only.

Is there a way to apply DiskID to a file, component or directory when using the heat tool?
Is it possible without using a transform?

**Comment by joeloff** [comment]
I'm 99% sure the answer is no. I did this years ago in order to split content into separate external CABs with WiX v3. The idea was that some CABs were unique to each MSI and others were shared between the installers. Something along these lines. The only difference between the .xslt files were the actual `DiskId` values as I recall.

```
    <xsl:template match="@* | node()">
        <xsl:copy>
            <xsl:apply-templates select="@* | node()"/>
        </xsl:copy>
    </xsl:template>

    <xsl:template match="wix:File">
        <xsl:copy>
            <xsl:attribute name="DiskId">
                <xsl:value-of select="2"/>
            </xsl:attribute>
            <xsl:apply-templates select="@* | node()"/>
        </xsl:copy>
    </xsl:template>
```
